test/functional/api/vim_spec.lua

```diff
@@ -1,6 +1,7 @@
 -- Sanity checks for vim_* API calls via msgpack-rpc
 local helpers = require('test.functional.helpers')
 local Screen = require('test.functional.ui.screen')
+local NIL = require('mpack').NIL
```

Comment:
I think its better to reexport NIL from helpers and do `local NIL = helpers.NIL` (so all imports from mpack are in helpers, except shada which deals with msgpack directly)